README.md

```diff
@@ -3,28 +3,36 @@
+
+You may use any of the Captain's Log commands anywhere inside of the Mojotech Organization. If you are using a private channel, you must go into the `channel integrations` and `add an app` called: `Captains Log`.
+
+### Quick Start
+NOTE: If you are a visual learner, then [this video tutorial](https://drive.google.com/file/d/1MbDeYRkuqv6KYRNZczgikMwOvgryeeLP/view?usp=sharing) is for you! Please beware: when installing the app, please do not install the dev version. Also, when using the commands, be sure to use `/captainslog` and not `/captainslogdev`.
+
+1. Okay, make sure that if you're in a private channel, `Captains Log` is installed as an app.
+2. Now, we need to make sure that you are in the chat of the channel where you want `Captains Log` to be configured in. Lets check to see if anyone on your team has already configured `/captainslog` for your current client and channel. Upon entering `/captainslog`, you should see a modal that says `Select Client for channel: <Your Channel name>`. If you do not see your client in this list, then click `x` on the top right of the modal. It is very important that you do not select anything that is not your client. If you see your client, go ahead and select it and click on the green `Submit` button. Else, lets create your client right now. Execute the command: `/captainslog new client`. Enter in the name of your client and click `Submit`. Now, go ahead and re-execute `/captainslog` and select your client.
+3. Lets create a new project. First, we need the URL for your github or gitlab repository. This can be found from copying the URL from the root of your repository's main/master branch.
+4. Go back to your slack channel from the previous step and execute the command: `/captainslog new project`. This should prompt you with a modal with 3 input fields. For the first field, select your client. For the second field, enter the name of your project. For the last field, paste in that URL that you have copied in the previous step. It is very important that the url starts with https and do not contain any query parameters. Go ahead and submit that. You should receive a message in your channel saying that new project has been added! You may repeat this step for however many projects need to be added. All releases will be handled the same for all projects under a client.
+5. Lets go ahead and configure webhooks for your repository. In github, you're going to need to be a repository admin. If you are not an admin, have an admin complete this step. From your repository, navigate to `Settings > Webhooks > Add webhook`. From here, enter in the endpoint: `https://captains-log-dev.fly.dev/webhook`, then click `Let me select individual events`. Scroll down, unselect `Pushes` and select `Releases`. Then scroll down and click the `Add webhook` button. If there are multiple projects with different repositories, then this step will need to be applied to each repository.
```

Comment:
is this right? the dev version?

Reply:
Yes, that `https://captains-log-dev.fly.dev/webhook` is the fly.io deployed url. I definitely goofed when making this. I was thinking of having asking for a CNAME alias for mojotech.com of captainslog.mojotech.com